com_dpfields/admin/helpers/dpfields.php

```diff
@@ -187,6 +197,14 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 						));
 					}
 
+					/*
+					 * On after field render
+					 * Event allow plugins to modfify the output of the prepared field
+					 */
+					if ($value)
```

Comment:
I think it should be the caller checking to see if the value is not empty and not left to the callee. what do you think?